### PR TITLE
Add Support for SwannOne Key Fob SWO-KEF1PA

### PR DESCRIPTION
**Pull request by qosmio**
The following is also needed.
## zigbee-shepherd-converters
https://github.com/qosmio/zigbee-shepherd-converters/commit/636d942debf1cd022e1254fd5c19b660ff518224
https://github.com/qosmio/zigbee-shepherd-converters/commit/e16ec72de444e85cca214e872cfb77f4ad7bf8d7

AND

## zigbee-shepherd
https://github.com/qosmio/zigbee-shepherd/commit/84860c1910c6f4a873fc4b1b670c74af18070bb8

Will be creating pull requests for those as well.

### CODE REVIEW

File: lib/extension/groups.js [hunk 1]
```diff
@@ -13,8 +13,11 @@ class Groups {
 
     onMQTTConnected() {
         this.mqtt.subscribe(`${settings.get().mqtt.base_topic}/bridge/group/+/remove`);
+        this.mqtt.subscribe(`${settings.get().mqtt.base_topic}/bridge/group/+/+/remove`);
         this.mqtt.subscribe(`${settings.get().mqtt.base_topic}/bridge/group/+/add`);
+        this.mqtt.subscribe(`${settings.get().mqtt.base_topic}/bridge/group/+/+/add`);
         this.mqtt.subscribe(`${settings.get().mqtt.base_topic}/bridge/group/+/remove_all`);
+        this.mqtt.subscribe(`${settings.get().mqtt.base_topic}/bridge/group/+/+/remove_all`);
```
Comment (Koenkk) [inline lib/extension/groups.js]: What is the reason for these changes?

**Comment by qosmio** [comment]
I use the the syntax "group/GROUP_NAME" so I can subscribe to the group topic in Home Assistant for certain triggers.

So, when I try to add/remove/remove_all devices  using "zigbee2mqtt/group/group/GROUP_NAME/add" it doesn't work.

I didn't realize this commit was included this pull request. Was going to open a new one.

File: lib/extension/deviceReceive.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const utils = require('../util/utils');
 const dontCacheProperties = [
     'action', 'button', 'button_left', 'button_right', 'click', 'forgotten', 'keyerror',
     'step_size', 'transition_time', 'action_color_temperature', 'action_color',
-    'action_group',
+    'action_group', 'armmode', 'panic',
```
Comment (Koenkk) [inline lib/extension/deviceReceive.js]: This shouldn't be needed as the properties for this device are only `action`.

Comment (qosmio) [reply lib/extension/deviceReceive.js]: You're right, I saw this after I had already updated my converter to delete armmode and replace it with action. I've updated it in on my end to take it out.

**Comment by Koenkk** [comment]
Closing this as device has been integrated.
